GentApp/ViewModels/CompanyViewModel.cs

```diff
@@ -38,15 +42,12 @@ public class CompanyViewModel : ViewModelBase {
 			}
 		}
 
-		private RelayCommand<string> _saveCompanyCommand;
-
-		public RelayCommand<string> SaveCompanyCommand {
-			get {
-				return _saveCompanyCommand = new RelayCommand<string>(name => {
-					UserViewModel.CurrentUser.Company = MyCompany;
-					UserViewModel.SaveUser(name);
-					RaisePropertyChanged(nameof(MyCompany));
-				});
+		public async Task SaveCompany() {
+			try {
+				await _companyService.Update(MyCompany);
+				MyCompany = await _companyService.GetMyCompany(UserViewModel.CurrentUser.Company.Id);
```

Comment:
wrm opvragen als ge upgedate company al hebt

Reply:
update company geeft de nieuwe objecten niet terug. Dit is om de nieuwe id's op te halen